### PR TITLE
fix: run acceptance tests for free-form properties for MongoDB

### PR DESCRIPTION
**Pull request by bajtos**
I noticed that the acceptance tests for MongoDB are skipping tests for free-form properties (strict mode turned off). This pull request is fixing the problem.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: acceptance/repository-mongodb/src/__tests__/mongodb.datasource.ts [hunk 1]
```diff
@@ -17,7 +17,6 @@ export const MONGODB_CONFIG: DataSourceOptions = {
   database: process.env.MONGODB_DATABASE || 'repository-tests',
 };
 
-export const MONGODB_FEATURES: CrudConnectorFeatures = {
+export const MONGODB_FEATURES: Partial<CrudConnectorFeatures> = {
   idType: 'string',
-  freeFormProperties: false,
```
Comment (nabdelgadir) [inline acceptance/repository-mongodb/src/__tests__/mongodb.datasource.ts]: Why is this removed?

Comment (emonddr) [reply acceptance/repository-mongodb/src/__tests__/mongodb.datasource.ts]: Probably because the default for freeFormProperties is true? @bajtos , can you confirm? thanks.

Comment (bajtos) [reply acceptance/repository-mongodb/src/__tests__/mongodb.datasource.ts]: Yes, `freeFormProperties` is true by default.

I would like connectors to try to support all features, including the ones added in the future. The `options` object should list only flags to disable features not supported by the connector.
